components/Navigation.js

```diff
@@ -2,28 +2,28 @@ import Link from "next/link";
 import { useRouter } from "next/router";
 import styled from "styled-components";
 import Image from "next/image";
-import BookmarkIcon from "assets/bookmarkicons/BookmarkPinIcon.svg";
+import BookmarkIcon from "assets/bookmarkicons/Bookmark-filled.svg";
 
 export default function Navigation() {
   const { pathname } = useRouter();
-  const isActive = (page) => pathname === page;
+  const $isActive = (page) => pathname === page;
 
   return (
     <StyledNavigationBar>
       <StyledLink href="/bookmarks">
-        <IconWrapper isActive={isActive("/bookmarks")}>
+        <IconWrapper $isActive={$isActive("/bookmarks")}>
           <StyledBookmarkIcon
```

Comment:
You should not need the `IconWrapper`. You can simply pass all the styles of your `IconWrapper` directly to your `StyledLink`. Try it out. 🙂 

```jsx
<StyledLink href="/bookmarks" $isActive={$isActive("/bookmarks")}>
     <StyledBookmarkIcon />
     My Emotions
</StyledLink>
```